spec/controllers/links_controller_spec.rb

```diff
@@ -15,4 +15,15 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe 'delete links' do
+    it 'handles deletion of links that have already been deleted' do
+      login_as_stub_user
+      delete :destroy, local_authority_slug: @local_authority.slug, service_slug: @service.slug, interaction_slug: @interaction.slug
+      expect(response).to have_http_status(302)
+      delete :destroy, local_authority_slug: @local_authority.slug, service_slug: @service.slug, interaction_slug: @interaction.slug
+      expect(response).to have_http_status(302)
+      expect(flash[:failed_action]).not_to be_present
+    end
```

Comment:
There's a weird quirk of controller tests/specs where trying multiple requests in a single test don't work properly.  But, I'm not sure we actually need the second request here?  Looking at the setup of the spec there isn't a `Link` instance yet, so the first request is already trying the deletion-of-a-link-that-doesn't-exist path, the second request seems redundant.